purchases/src/test/java/com/revenuecat/purchases/common/offerings/OfferingsManagerTest.kt

```diff
@@ -407,62 +407,6 @@ class OfferingsManagerTest {
         verify(exactly = 1) { cache.clearOfferingsCacheTimestamp() }
     }
 
-    @Test
-    fun `getOfferings marks offerings cache timestamp upon success`() {
```

Comment:
These tests don't make sense now, and we have other tests verifying we are caching offerings.